R/generics.R

```diff
@@ -483,6 +483,26 @@ setGeneric("plotChord", function(obj, addSelfLinks = FALSE, addRetMzPlots = TRUE
 #' @template generics
 setGeneric("plotChroms", function(obj, ...) standardGeneric("plotChroms"))
 
+#' @templateVar func getTICs
+#' @templateVar desc gets total ion chromatogram(s).
+#' @template generics
+setGeneric("getTICs", function(obj, ...) standardGeneric("getTICs"))
```

Comment:
I'm sort of trying to keep the generics in this file section alphabetically sorted.

Reply:
Now I know, done.